app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
``` 
2 lệnh này khcas gì nhau nhỉ?

Reply:
Nếu dùng validates :user thì chỉ cần object được liên kết tồn tại trong bộ nhớ là validation sẽ valid. 
Còn validates :user_id thì rails sẽ phải check xem user_id này có tồn tại User nào tương ứng trên database không thì mới cho valid (nếu dùng validates :user mà user là nil nhưng user_id khác nil thì Rails sẽ check như khi validates :user_id).